src/ReplenishFunction.ts

```diff
@@ -87,6 +120,13 @@ async function defaultReplenishFunction(
       );
       transactionHashes.push(txHash);
     } else {
+      notify(
+        'Not possible to replenish the worker due to manager balance too low',
+        `Worker address: ${relayServer.workerAddress}`,
+        `Manager address: ${relayServer.managerAddress}`,
```

Comment:
shouldnt be a good idea to include the worker balance?

Reply:
Yes, it is, thank you

Reply:
Updated and also I changed the format of the message.